### ISSUE TITLE
special request: mapping ability in the distrho host params module

### ISSUE THREAD

**Issue by offthesky** [DISTRHO/Cardinal#110]
loving cardinal! being able to map host params to various knobs or buttons that dont have a cv in would be super amazing!

**Comment by falkTX** [comment]
How do you suppose we could do that?

Maybe a module that can map alike MIDI CC, but for CV sources? So we can automate any regular parameter with a CV cable..?

**Comment by offthesky** [comment]
yes, that would be amazing!  a rough though i had in my head (pardon my bad photoshops)
![cardinalmapper idea](https://user-images.githubusercontent.com/95579081/148701535-85aa49e3-10a0-40cb-9ffa-4e7fdaa714fe.jpg)

**Comment by falkTX** [comment]
Forgot this ticket was still open.
This specific feature is now possible using "Host Parameters Map" module, coming in 22.06 today
